.eslintrc.json

```diff
@@ -19,7 +19,14 @@
   "overrides": [
     {
       "files": ["build/**/*.js", "main/**/*.js", "test/**/*.js", "*.js"],
+      "parserOptions": {
+        "sourceType": "commonjs"
```

Comment:
I think auto-detection of source type does not work when using the TypeScript parser.

See https://github.com/silvermine/eslint-config-silvermine/issues/30#issuecomment-454897158 and https://github.com/eslint/typescript-eslint-parser/pull/583